web/views.py

```
@@ -6913,7 +6913,7 @@ def get_twitter_client():
     return tweepy.API(auth)
 
 
-@user_passes_test(social_media_manager_required)
+@user_passes_test(social_media_manager_required, login_url="/accounts/login/")
 def social_media_dashboard(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `social_media_dashboard`

(ANN201)